src/decorator/live-decorator.ts

```diff
@@ -31,6 +33,14 @@ export class KalturaLiveEngineDecorator implements KalturaPlayerTypes.IEngineDec
     }
 
     dispatchEvent(event: any): any {
+        if (event.type === EventTypes.METADATA) {
+            this._plugin.handleTimedMetadata(event);
+        }
+        if (event.type === EventTypes.SEEKED) {
+            // check _isOnLiveEdge property after 'SEEKING' event happened
```

Comment:
the comment is irrelevant any more no?

Reply:
relevant
after "SEEKING" happenED we check the state. It more like explanation how the plugin communicate with player.
Might be content of comment can be changed, but the goal - should be same

Reply:
If you want I can change the comment to:
`// check _isOnLiveEdge property each time when 'SEEKED' event happens`

Reply:
I ok with any comments. just thought it was related to the timeout.